common/src/main/java/com/wynntils/features/user/TradeMarketAutoOpenChatFeature.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.mc.event.ScreenClosedEvent;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.wc.event.ChatMessageReceivedEvent;
+import com.wynntils.wc.utils.WynnUtils;
+import java.util.regex.Pattern;
+import net.minecraft.client.gui.screens.ChatScreen;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class TradeMarketAutoOpenChatFeature extends UserFeature {
+
+    // Type the price in emeralds or type 'cancel' to cancel:
+    // Type the amount you wish to buy or type 'cancel' to cancel:
+    // Type the item name or type 'cancel' to cancel:
+    private static final Pattern TYPE_TO_CHAT_PATTERN = Pattern.compile("§6Type the .* or type 'cancel' to cancel:");
```

Comment:
In general, I like regexp patterns to include `^` and `$`, that makes sure we match the whole line, which makes the risk lower that we match on a partial message, potentially sent by a user.